auth-web/src/views/auth/IncorpOrRegisterView.vue

```diff
@@ -37,18 +37,21 @@
         </template>
         <!-- Not Authenticated -->
         <template v-else>
-          <v-btn large color="#fcba19" @click="login()" class="mt-5">
+          <v-btn large color="#fcba19" @click="emitLogin()" class="mt-5">
             Log in with BC Services Card
           </v-btn>
           <p class="my-5">New to BC Registries? <a @click="emitAccountDialog()" class="create-account-link">
             <u>Create a BC Registries Account</u></a>
           </p>
         </template>
-        <LearnMoreButton />
+        <learn-more-button
+        :redirect-url="'GO SOMEWHERE'"/>
```

Comment:
Not ready for MVP. Also on line 52.
Also should indent or move to prev line.

Reply:
Sorry, another placeholder 🤦 . This won't make it to MVP 100% but i think to be safe i will put in another help link until the proper one is provided. I think this doc or static page is currently a wip.

Reply:
Updated in next commit!